Chat/DataStorage/IContosoEventsData.cs

```diff
@@ -4,6 +4,12 @@
 
 namespace Chat
 {
+	public interface IEventsStore
+	{
+		// [event id -> moderator id] 
+		Dictionary<string, ACSEvent> Store { get; }
```

Comment:
why do we need a map of event to moderator?

Reply:
whoops, that comment is incorrect

Reply:
it should read [event id -> event object]